Makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: start
+start:
+	./scripts/init.sh start-frequency-instant
+
```

Comment:
make file to simplify interaction with Frequency
make start will start dev mode with instant-sealing